From 70b4b21503c51e8aa74f3d07813569d4155349f1 Mon Sep 17 00:00:00 2001
From: Sebastian <sebix@sebix.at>
Date: Sat, 17 Aug 2024 11:02:16 +0200
Subject: [PATCH] fix exit codes for help and version

command line options `-h` (help) and `-V` (version) resulted in an exit code 1 (error), not 0 (no error)
---
 r2t.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/r2t.c b/r2t.c
index a73eb9d..af663cf 100644
--- a/r2t.c
+++ b/r2t.c
@@ -333,12 +333,12 @@ int main(int argc, char *argv[])
 
 			case 'V':
 				version();
-				return 1;
+				return 0;
 
 			case 'h':
 			default:
 				usage();
-				return 1;
+				return 0;
 		}
 	}
 
